### DISCUSSION TITLE
What is internal storage format for Publii

### DISCUSSION THREAD

**Discussion by epogrebnyak** [GetPublii/Publii#1427] [Help] [answered]
Does Public store content as markdown? Could not find an answer in the docs.

**Comment by internettips** [comment, marked as answer]
@epogrebnyak 

Yes, sort of. You can create content pages using one of 3 different editors: WYSIWYG, or Block, or Markdown. Refer to 'Choosing an Editor' on: https://getpublii.com/docs/adding-editing-and-deleting-posts.html

**Reply by epogrebnyak** [reply]
Thank you!

**Comment by candidexmedia** [comment]
It stores all posts, tags, and authors in the db.sqlite database file found in the input folder of your site's directory. You can view the contents of the database using apps like [DB Browser for SQLite](https://sqlitebrowser.org/) and [SQLite Studio](https://sqlitestudio.pl/).

The saved format of the post inside of the db will depend on the Editor you select.

**Reply by epogrebnyak** [reply]
Thank you, that also answers the question if the source is available as plain files, as in Hugo or Jekyll, Publii has a different design.
